docs/src/paradox/01-introduction/what-is-knora.md

```diff
@@ -19,51 +19,101 @@ License along with Knora.  If not, see <http://www.gnu.org/licenses/>.
-requires a common, flexible data representation and storage technology
-capable of performing queries across large quantities of heterogeneous
-data, organised according to project-specific data structures that
-cannot be known in advance. It also requires a convenient,
-storage-independent way for Virtual Research Environments (VREs) and
-automated data-processing software to access, query, and add to this
-data.
-
-To solve the data representation and storage problem, Knora represents
-humanities data as
-[RDF](http://www.w3.org/TR/2014/NOTE-rdf11-primer-20140624/) graphs,
-using [OWL](http://www.w3.org/TR/2012/REC-owl2-primer-20121211/)
-ontologies that express abstract, cross-disciplinary commonalities in
-the structure and semantics of research data. Each project using Knora
-extends these abstractions by providing its own project-specific
-ontology, which more specifically describes the structure and semantics
-of its data. Existing non-RDF repositories can readily be converted to
-an RDF format based on the proposed abstractions. This design makes it
-possible to preserve the semantics of data imported from relational
-databases, XML-based markup systems, and other types of storage, as well
-as to query, annotate, and link together heterogeneous data in a unified
-way. By offering a shared, standards-based, extensible infrastructure
-for diverse humanities projects, Knora also deals with the issue of
-conversion and migration caused by the obsolescence of file and data
-formats in an efficient and feasible manner.
-
-To solve the access problem, Knora offers generic HTTP-based APIs. These
-allow applications to query and work with data in terms of the concepts
-expressed by the Knora ontologies, without dealing with the complexities of the
-underlying storage system. This approach also allows Knora to provide
-features that are not part of the RDF standards, such as access control and
-automatic versioning of data, and to act as a gateway to other sorts of
-repositories, including non-RDF repositories.
-
-Knora uses a high-performance media server, called [Sipi](http://www.sipi.io/),
-for serving and converting binary media files such as images and video. Sipi can
-efficiently convert between many different formats on demand, preserving
-embedded metadata, and implements the [International Image
-Interoperability Framework (IIIF)](http://iiif.io/).
+a content management system for the long-term preservation and reuse of
+humanities data. It is designed to accommodate data with a complex internal
+structure, including data that could be stored in relational databases.
+
+A key problem in the long-term preservation of data is the multitude of
+data formats in use, many of which are proprietary and quickly become obsolete.
+It is not practical to maintain all the software that was used to create and
+read old data files.
+
+Instead of preserving a multitude of different data formats, Knora supports
```

Comment:
I think you should stress the difference between a classical digital archive and Knora: In a digital archive, contents are preserved by using standard archival formats like PDF/A but they are not directly accessible (https://en.wikipedia.org/wiki/Open_Archival_Information_System). The contents in a digital archive are only searchable via metadata. Once you have found something, it is handed over to you.

When it comes to databases, these are stored in a non functional state, e.g., a relational database is serialized to XML.

Knora on the other hand preserves data by keeping it alive. It is a keep-alive archive. You can import a relational database into Knora and still query it. Of course, we do not emulate the original SQL queries.

Reply:
I tried to describe that here: http://dhlab.unibas.ch/documents/schweizer//archiving_2014.pdf

It is outdated because it was written before Knora came into existence. But I think the basic thought is still valid.